public/app/features/dashboard-scene/scene/layout-rows/RowItem.tsx

```diff
@@ -237,3 +270,64 @@ export function RowHeightSelect({ row }: { row: RowItem }) {
     />
   );
 }
+
+export function RowRepeatSelect({ row, dashboard }: { row: RowItem; dashboard: DashboardScene }) {
+  const { layout, $behaviors } = row.useState();
+
+  let repeatBehavior: RowItemRepeaterBehavior | undefined = $behaviors?.find(
+    (b) => b instanceof RowItemRepeaterBehavior
+  );
+  const { variableName } = repeatBehavior?.state ?? {};
+
+  const isAnyPanelUsingDashboardDS = layout.getVizPanels().some((vizPanel) => {
```

Comment:
This could probably be addressed in a different PR, I recall this message being shown in other places as well, but since we now support dashboardDS queries in mixedDS we should probably also check if the datasource is mixed and if any of those subqueries are dashboardDS ones

```
runner.state.datasource?.uid === MIXED_DATASOURCE_NAME &&
      runner.state.queries.some((query) => query.datasource?.uid === SHARED_DASHBOARD_QUERY)
```